content/guides/saml.md

```diff
@@ -0,0 +1,108 @@
+* If **sn** and **givenName** are provided, they will be used to update the user’s name in their Datadog profile. 
+
+## Additional Features Available On Request
+The following features are available on request. If you would like access to them, contact support and describe your scenario. The feature will be available on  your account after some configurations on our side.
+
+### Just in Time Provisioning (JIT Provisioning)
+Normally users must be invited to Datadog, even for organizations with SAML enabled. If a user that has not been invited to Datadog logs in via an Org's IdP, the SAML assertion will be validated, but they will be redirected to a SAML Error page.
+
+Some organizations might not want to have to invite all of their users to Datadog. If the organization has SAML enabled, and the saml_autocreate_user feature flag is enabled, Datadog will automatically create a user in response to a valid SAML assertion that contains a user handle that doesn't exist yet. This way anyone with a valid login on the organization's IdP can use Datadog without needing an invite.
+
+It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
+
+### IdP Initiated Login
+
+The normal workflow is that when the Datadog url is loaded, the browser is redirected to the customer IdP, user types in credentials, then the IdP redirects back to Datadog. Some IdPs have the ability to send an assertion directly to Datadog without first getting an AuthnRequest (IdP Initiated Login). 
+
+In the normal setup, we won't know which org the assertion came from and this will result in an error page with a message saying that SAML Response is missing "InResponseTo" attribute. 
+
+After enabling the feature (and waiting for caches to clear) the customer will need to get a new version of the SP Metadata, which will have a different, org-specific AssertionConsumerService endpoint to send assertions to.
+
+### SAML Strict Mode
+For an org that has the saml_strict_mode feature flag enabled, users can not log in using a username and password on the login page - they must log in via their org's SAML IdP.
+
+Normally users who are invited to an org using SAML will not have a password, so they can't use the login page anyway. However, there are possibilities where a user at a SAML org can have a password
+
+* The user was created before the Org enabled SAML
+* Support added a temporary password and the user reset it In these cases (or in any case where a user has a password in the postgres database) a user can log in either with SAML (via the IdP) or with the login page using their Datadog username and password.
```

Comment:
This might benefit from a bit of a re-write.  The original doc was written with internal, support users in mind.  For instance, I don't think we should mention postgres database in a public document.